utils/docker/run-doc-update.sh

```diff
@@ -32,20 +32,33 @@
 
 set -e
 
-ORIGIN="https://${GITHUB_TOKEN}@github.com/pmem-bot/libpmemobj-cpp"
-UPSTREAM="https://github.com/pmem/libpmemobj-cpp"
+source `dirname $0`/valid-branches.sh
+
+BOT_NAME="pmem-bot"
+USER_NAME="pmem"
+REPO_NAME="libpmemobj-cpp"
+
+ORIGIN="https://${GITHUB_TOKEN}@github.com/${BOT_NAME}/${REPO_NAME}"
+UPSTREAM="https://github.com/pmem/${REPO_NAME}"
+# master or stable-* branch
+TARGET_BRANCH=${TRAVIS_BRANCH}
+VERSION=${TARGET_BRANCHES[$TARGET_BRANCH]}
```

Comment:
this will end badly when TRAVIS_BRANCH is empty (eg on gha), see my recent fix to pmdk

Reply:
Done.

Reply:
This still doesn't look correct.

https://github.com/pmem/pmdk/commit/eeb8fd86c6c72f4607a22a1e9d9f7da31be8af2a